frontend/packages/dev-console/src/components/helm/HelmReleaseList.tsx

```diff
@@ -0,0 +1,116 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { coFetchJSON } from '@console/internal/co-fetch';
+import { Table, TextFilter } from '@console/internal/components/factory';
+import { SortByDirection } from '@patternfly/react-table';
+import { CheckBoxes } from '@console/internal/components/row-filter';
+import { FirehoseResult, getQueryArgument } from '@console/internal/components/utils';
+import { HelmRelease, HelmFilterType } from './helm-types';
+import { helmRowFilters, getFilteredItems, useDeepCompareMemoize } from './helm-utils';
+import HelmReleaseHeader from './HelmReleaseHeader';
+import HelmReleaseRow from './HelmReleaseRow';
+
+interface HelmReleaseListProps {
+  namespace: string;
+  secrets?: FirehoseResult;
+}
+
+const HelmReleaseList: React.FC<HelmReleaseListProps> = ({ namespace, secrets }) => {
+  const [releases, setReleases] = React.useState([]);
+  const [filteredReleases, setFilteredReleases] = React.useState([]);
+  const [fetched, setFetched] = React.useState(false);
+
+  const memoizedSecrets = useDeepCompareMemoize(secrets.data);
+
+  React.useEffect(() => {
+    let ignore = false;
+
+    const activeFilters =
+      getQueryArgument('rowFilter-helm-release-status') &&
+      getQueryArgument('rowFilter-helm-release-status').split(',');
+
+    const fetchHelmReleases = async () => {
+      const res: HelmRelease[] = await coFetchJSON('/api/console/helm/list');
```

Comment:
@rohitkrai03 we've updated `/api/console/helm/list` API path to `/api/helm/releases`, please update here.

Reply:
ref : https://github.com/openshift/console/blob/60ab162ca1f79d7c5780ed0adf8ef5b7a2da3e8c/pkg/server/server.go#L303

Reply:
@akashshinde Sure.

Reply:
Can you also update your PR description https://github.com/openshift/console/pull/3826 with new API endpoints?